JEMS-CSE416/JEMS
# Responsive Map Cards on home screen

**Updates**
- Added responsiveness for map cards displayed on home screen for screens of 1250, 1500px, and over 1500px 

**Setup:**
```
git checkout main
git fetch
git checkout home_screen
cd client
npm start
```

**Testing**
1. Go to browser
2. Click Get Started Button or navigate to `/home`
3. You should see a home screen with maps cards that respond favorably for screens over 1000px
